### ISSUE TITLE
 [DEPRECATION] `rack-throttle` is deprecated.  Please use consider using `rack-attack` https://github.com/rack/rack-attack instead.

### ISSUE THREAD

**Issue by jplitza** [pglombardo/PasswordPusher#1471]
## 🐛 Bug Report

Since updating to v1.31.5, calling `rake daily_expiration` or `rake delete_expired_and_anonymous` produces the following output on stderr:

```
[DEPRECATION] `rack-throttle` is deprecated.  Please use consider using `rack-attack` https://github.com/rack/rack-attack instead.
[DEPRECATION] `rack-throttle` is deprecated.  Please use consider using `rack-attack` https://github.com/rack/rack-attack instead.
[DEPRECATION] `rack-throttle` is deprecated.  Please use consider using `rack-attack` https://github.com/rack/rack-attack instead.
[DEPRECATION] `rack-throttle` is deprecated.  Please use consider using `rack-attack` https://github.com/rack/rack-attack instead.
```

## 🔬 How To Reproduce

1. Use v1.31.5
2. Call `rake daily_expiration` or `rake delete_expired_and_anonymous`

### Environment

Where are you running/using Password Pusher?

- [ ] pwpush.com
- [ ] Docker
  - [ ] pwpush-ephemeral
  - [ ] pwpush-mysql
  - [ ] pwpush-postgres
  - [ ] custom image
- [ ] Heroku
- [ ] Digital Ocean
- [ ] Microsoft Azure
- [ ] Google Cloud
- [ ] AWS
- [x] Source Code
- [ ] Other (please specify)

If applicable, what version of Password Pusher?

v1.31.5

## 📈 Expected behavior

No output on stderr

**Comment by pglombardo** [comment]
Thanks for reporting!  This will be fixed in the next release.

**Comment by jplitza** [comment]
Glad you won the fight against dependabot. ;-)

I guess this issue will stay open until you (or someone more knowledgeable about Rails than me) managed to migrate from rack-attack to rack-throttle.

**Comment by pglombardo** [comment]
Yeah rack-throttle is still a great solution - only that it's not developed anymore.  When I get some time (or someone from the community) we'll change it out.  But for now I think it's low priority.

**Comment by pglombardo** [comment]
Although this issue was technically fixed last year, I never closed the issue.  🤦 

Now July 2024, rack-throttle has been removed and replaced with rack-attack in #2324

The change-over should be transparent for users.

Hope all is well @jplitza.
